pkgs/build-support/fetchurl/default.nix

```diff
@@ -124,6 +124,8 @@ if (!hasHash) then throw "Specify hash for fetchurl fixed-output derivation: ${s
 
   inherit curlOpts showURLs mirrorsFile impureEnvVars postFetch downloadToTemp executable;
 
+  userAgent = "Nix ${builtins.nixVersion} on nixpkgs ${stdenv.lib.nixpkgsVersion}";
```

Comment:
None, but many of us aren't on NixOS :P
On Fri, Sep 2, 2016 at 11:31 Vladimír Čunát <notifications@github.com>
wrote:

> In pkgs/build-support/fetchurl/default.nix
> <https://github.com/NixOS/nixpkgs/pull/17757#discussion_r77365738>:
>
> > @@ -124,6 +124,8 @@ if (!hasHash) then throw "Specify hash for fetchurl fixed-output derivation: ${s
> >
> >    inherit curlOpts showURLs mirrorsFile impureEnvVars postFetch downloadToTemp executable;
> >
> > +  userAgent = "Nix ${builtins.nixVersion} on nixpkgs ${stdenv.lib.nixpkgsVersion}";
>
> I think it would be enough to put nixos version in there, e.g. 16.09 ATM,
> or are there any expectations about the version?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/NixOS/nixpkgs/pull/17757/files/75b22922b8f285cf0e36ab900a807e3ca5f4ca6f#r77365738>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAKP03oU-w8_fc6yA-SfjoQgse6O4cTks5qmEE2gaJpZM4Jkbay>
> .
>